tests/ha/single_pod_failure/test_server_pod_failure.py

```diff
@@ -174,15 +176,34 @@ def test_degraded_reads_safe_server_pod_shutdown(self):
         assert_utils.assert_true(resp[0], resp[1])
         LOGGER.info("Step 2: Performed READs and verified DI on the written data")
 
-        LOGGER.info("Step 3: Shutdown random server pod safely by making replicas=0 and verify "
-                    "cluster & remaining pods status")
+        num_replica = 0
+        LOGGER.info("Get pod to be deleted")
```

Comment:
Is is better to specify name of pod - data or server or data/server

Reply:
We are printing the pod name on line 186.